CMakeLists.txt

```diff
@@ -15,9 +15,20 @@ cet_set_compiler_flags(DIAGS VIGILANT
  -Wno-unused-parameter -Wno-type-limits -Wno-ignored-qualifiers -Wno-unused-but-set-parameter -Wno-extra -Wno-non-virtual-dtor -Wno-implicit-fallthrough -Wno-deprecated-copy
 )
 
+if(DEFINED Offline_UPS_QUALIFIER_STRING)
+  string(FIND ${Offline_UPS_QUALIFIER_STRING} "trig" TRIG_BUILD)
+  # TRIG_BUILD will be -1 if trig qualifier not found
+  if(${TRIG_BUILD} GREATER_EQUAL 0)
+    set(WITH_G4 FALSE)
+  else()
+    set(WITH_G4 TRUE)
+  endif()
+endif()
+
 message("WITH_G4 SET TO ${WITH_G4}")
 
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/..)
+message("Creating link from ${CMAKE_CURRENT_SOURCE_DIR} to ${CMAKE_CURRENT_SOURCE_DIR}/Offline to satisfy includes")
```

Comment:
Basically the problem here is that "Offline" is the name of the Git repository, but if you archive the contents of the repository and extract in a random directory, it is not guaranteed that you will have all of the Offline packages in a directory named "Offline". Since the convention was adopted to include "Offline/" in all of the include statements and fhicl documents, this can cause issues when you try to build from such an archive (as Spack does). Adding a symlink to the current directory with the proper name avoids this issue with the added benefit of not having to add the upper-level directory to the include path (which could contain other things).

Reply:
For example: download https://github.com/Mu2e/Offline/archive/refs/tags/v10_26_00.tar.gz and extract it

Reply:
Is the download and untar of this tar file a step that you have control over or does it happen deep with in cmake/spack???

If you have control, why not clone and checkout rather than download the tar.gz file?  Or why not rename after untar?

Reply:
@eflumerf a gentle reminder that I am wating on your followup to these comments.

Reply:
It happens within the Spack build, and I could not find a way to unwind in a subdirectory of my choosing. The configure step is automatically called from the automatically-chosen untar dir, so I have no control over that phase. This is why I went with the symlink, as that is the only thing that is explicitly under CMake's control.

Reply:
I will approve and merge.  As we learn how to use spack I would really like to get rid of this sort of hack.

My past experience with symlinks is that people start writing include files using the symlink and not the natural path.  I don't know if that's a thing in this case bue.